### PR TITLE
Remove redundant mentions of "CENNZnet"

### LINKED ISSUE #4: Remove redundant mentions of "CENNZnet" on the header
Screen Shot 2022-05-16 at 09 07 00

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

 - closes #4 
 - update imports/exports to use `@polkadot/*` instead of `@/libs/*`
 - restore `storage` route (chainstate)

### CODE REVIEW

File: polkadot/packages/react-query/src/Chain.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ function Chain ({ children, className = '', label }: Props): React.ReactElement<
 
   return (
     <div className={className}>
-      {label || ''}{systemChain || t('Unknown')}{children}
+      {label || ''}{systemChain?.replace("CENNZnet ", "") || t('Unknown')}{children}
```
Comment (ken-futureverse) [inline polkadot/packages/react-query/src/Chain.tsx:22]: why do we have to touch the `react-query` to replace the `CENNZnet`?

Comment (aidan-starke) [reply polkadot/packages/react-query/src/Chain.tsx:22]: systemName comes from `api.rpc.system.name()` so it's either overriding it in `Api` or editing `react-query`,
same as systemChain which comes from `api.rpc.system.chain()`

**Comment by aidan-starke** [comment]
covered in #5

**Comment by aidan-starke** [comment]
covered in #5

**Pull request closed without merging**